hic_qc.py

```diff
@@ -917,6 +1009,10 @@ def parse_args():
     parser.add_argument('--mq_stats', nargs='+', default=[0, 1, 10, 20, 30, 40], help='List of min MQ scores to calculate RP stats for (Default: %(default)s)')
     parser.add_argument('--edist_stats', nargs='+', default=[100, 10, 5, 3, 1, 0], help='List of max edist scores to calculate RP stats for (Default: %(default)s)')
     parser.add_argument('--version', action='version', version=__version__)
+    parser.add_argument('--thresholds', default='./collateral/thresholds.json', 
```

Comment:
This should be determined relative to the python script using the `__file__` property

Reply:
Done